editor/enable-tracking-prompt/index.js

```diff
@@ -59,7 +59,10 @@ export function EnableTrackingPrompt( props ) {
 }
 
 export default connect(
-	undefined,
+	( state ) => { // eslint-disable-line no-unused-vars
+		return {
+			bumpStat,
+		};
+	},
```

Comment:
This needs a better solution (probably a higher-order component), but it was the easiest way I could think of to allow mocking the `bumpStat` function.